app.js

```diff
@@ -63,7 +63,9 @@ async function create_user(){
     window.location.replace(redirect);
 }
 
+// declaring global varibles to be used across functions
 var activeMovieID = 1;
+var activeMovieName = "Knives Out";
```

Comment:
I assume these are placeholder values that get over written?

Reply:
These are where we create the variables for their use, the values they take are essentially place holders. They will change within Rafiq's method when it updates the page to show a new movie